spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/actions/TestCreateActions.java

```diff
@@ -490,6 +490,11 @@ public void testSparkTableReservedProperties() throws Exception {
     Assert.assertEquals("Unexpected format", "iceberg/parquet", table.properties().get("format"));
     Assert.assertNotEquals("No current-snapshot-id found", "none", table.properties().get("current-snapshot-id"));
     Assert.assertTrue("Location isn't correct", table.properties().get("location").endsWith(destTableName));
+
+    Assert.assertNotNull("Missing format-version", table.properties().get("format-version"));
```

Comment:
Not asserting the value being 1 in case we move the default table format-version to 2 (this test may simply break then)

Reply:
I think it's fine to assert the value is 1. That's the current expected behavior (format-version is 1 for tables created by Spark).

If / when we update the default to 2, we'll update the test.